djstripe/models.py

```diff
@@ -45,12 +45,41 @@ def convert_tstamp(response, field_name=None):
 
 
 class StripeObject(TimeStampedModel):
+    # This must be defined in descendants of this model/mixin
+    # e.g. "Event", "Charge", "Customer", etc.
+    stripe_api_name = None
 
     stripe_id = models.CharField(max_length=50, unique=True)
 
-    class Meta:
+    class Meta(object):
```

Comment:
I think I'll just change it back. Maybe someday we can change them all at one time. But might as well keep them all in one style or the other